operator/v1/types_network.go

```diff
@@ -388,6 +393,65 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+// GatewayMode holds the node gateway mode
+type GatewayMode string
+
+const (
+	// GatewayModeShared indicates OVN shares a gateway interface with the node
+	GatewayModeShared GatewayMode = "shared"
+	// GatewayModeLocal indicates OVN creates a local NAT-ed interface for the gateway
+	GatewayModeLocal GatewayMode = "local"
+)
+
+// GatewayConfig holds node gateway-related parsed config file parameters and command-line overrides
+type GatewayConfig struct {
```

Comment:
It's possible some of these options are not valid/used currently in OVN-K. Need to double check.

Reply:
Seems like all the options are still in play,